web/src/app/alerts/components/AlertDetails.tsx

```diff
@@ -189,14 +221,14 @@ export default function AlertDetails(props) {
 
             if (completed) return 'Escalating...'
 
-            const hourTxt = parseInt(hours)
-              ? `${hours} hour${parseInt(hours) === 1 ? '' : 's'} `
+            const hourTxt = parseInt(`${hours}`)
+              ? `${hours} hour${parseInt(`${hours}`) === 1 ? '' : 's'} `
               : ''
-            const minTxt = parseInt(minutes)
-              ? `${minutes} minute${parseInt(minutes) === 1 ? '' : 's'} `
+            const minTxt = parseInt(`${minutes}`)
+              ? `${minutes} minute${parseInt(`${minutes}`) === 1 ? '' : 's'} `
               : ''
             const secTxt = `${seconds} second${
-              parseInt(seconds) === 1 ? '' : 's'
+              parseInt(`${seconds}`) === 1 ? '' : 's'
             }`
```

Comment:
it looks like all of these are being returned as numbers from `Countdown` so I think we can forego the `parseInt` calls altogether

```
const hourTxt = hours ? `${hours} hour${hours === 1 ? '' : 's'} ` : ''
const minTxt = minutes ? `${minutes} minute${minutes === 1 ? '' : 's'} ` : ''
const secTxt = `${seconds} second${seconds === 1 ? '' : 's'}`
```